lychee-lib/src/client.rs

```diff
@@ -494,10 +535,7 @@ impl Client {
     /// A better approach would be to download the file through the API or
     /// clone the repo, but we chose the pragmatic approach.
     async fn check_github(&self, uri: GithubUri) -> Status {
-        let client = match &self.github_client {
-            Some(client) => client,
-            None => return ErrorKind::MissingGitHubToken.into(),
-        };
+        let Some(client) = &self.github_client else { return ErrorKind::MissingGitHubToken.into() };
```

Comment:
Oh, I didn't know that this pattern is valid.

Reply:
this is very new.